<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Biblioteka multimediów v1.5</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>30</height>
   </size>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>901</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Kolekcja płyt CD</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>701</width>
        <height>451</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Autor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tytuł</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Czas</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Wydawnictwo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Utworów</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ilość</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>40</y>
        <width>171</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Nowa płyta</string>
      </property>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>128</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>128</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tytuł</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>80</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Wykonawca</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Wydawnictwo</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>128</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Czas trwania</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
       <property name="placeholderText">
        <string>mm:ss</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Liczba utworów</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>128</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>230</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Dodaj pozycję</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>310</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuń zaznaczoną pozycję</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>10</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Szukaj płyty po autorze lub tytule</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Kolekcja kaset wideo</string>
     </attribute>
     <widget class="QTableWidget" name="kaswideo_tab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>701</width>
        <height>451</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Autor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tytuł</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Czas</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Wydawnictwo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Format</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Jakość</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ilość</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>40</y>
        <width>171</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Nowa kaseta wideo</string>
      </property>
      <widget class="QLineEdit" name="kas_autor">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>128</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="kas_tytul">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>128</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tytuł</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Autor</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Wydawnictwo</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="kas_wydaw">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>128</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Czas trwania</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="kas_czas">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
       <property name="placeholderText">
        <string>mm:ss</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Format kasety</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>230</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Dodaj pozycję</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>340</y>
        <width>171</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Dodatkowe informacje o D-VHS</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Jakość (kbps)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="kas_jakosc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Dodaj pozycję</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>310</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuń zaznaczoną pozycję</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>10</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Szukaj kasety po autorze lub tytule</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Kolekcja kaset audio</string>
     </attribute>
     <widget class="QTableWidget" name="kasaudio_tab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>701</width>
        <height>451</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Autor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tytuł</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Czas</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Wydawnictwo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cena</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ilość</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>40</y>
        <width>171</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Nowa kaseta audio</string>
      </property>
      <widget class="QLineEdit" name="kasa_autor">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>128</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="kasa_tytul">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>128</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tytuł</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Wykonawca</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Wydawnictwo</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="kasa_wydaw">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>128</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Czas trwania</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="kasa_czas">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
       <property name="placeholderText">
        <string>mm:ss</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Cena za kasetę</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="kasa_cena">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>128</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>230</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Dodaj pozycję</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>310</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuń zaznaczoną pozycję</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>10</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Szukaj kasety po autorze lub tytule</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
